website/views/shared/tasks/lists.jade

```diff
@@ -24,8 +24,8 @@ script(id='templates/habitrpg-tasks.html', type="text/ng-template")
 
           // Add New
           form.task-add(name='new{{list.type}}form', ng-hide='obj._locked', ng-submit='addTask(obj[list.type+"s"],list)')
-            textarea(rows='6', ng-model='list.newTask', placeholder='{{list.placeHolderBulk}}', ng-if='list.bulk', ui-keydown='{"meta-enter ctrl-enter":"addTask(obj[list.type+\'s\'],list)"}', required)
-            input(type='text', ng-model='list.newTask', placeholder='{{list.placeHolder}}', ng-if='!list.bulk', required)
+            textarea(rows='6', task-focus='list.bulk && list.focus', ng-model='list.newTask', placeholder='{{list.placeHolderBulk}}', ng-if='list.bulk', ui-keydown='{"meta-enter ctrl-enter":"addTask(obj[list.type+\'s\'],list)"}', required)
```

Comment:
I didn't either. I was planning to add my own (though I really didn't want to), because there didn't seem to be any other way to get `focus()` to work. But this was the first directive in the file, so it worked out!